plans/acceptance/pe_server.pp

```diff
@@ -4,6 +4,11 @@
 #
 # @example
 #   pe_event_forwarding::acceptance::pe_server
+#
+# @param version
+#   PE version
+# @param pe_settings
+#   Password needed to install PE
```

Comment:
This password will be the console password for the admin user to log in with once installation is complete. It's not needed to install PE itself.